### PR TITLE
docsp-32913 - v1.3 release

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-visual-studio-extension/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-32913
Staging - https://docs-mongodbcom-staging.corp.mongodb.com/visual-studio-extension/docsworker-xlarge/docsp-32913-v1.3/whats-new/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

File: snooty.toml [hunk 1]
```diff
@@ -13,15 +11,14 @@ product-source-repo = "https://github.com/mongodb/mongo-csharp-analyzer"
...
 product-settings-file-constant = "mongodb.analyzer.json"
 product-jira = "https://jira.mongodb.org/browse/VS"
 
 query-api = "MongoDB Query API"
 
-driver-long = "MongoDB .NET driver"
-driver-short = ".NET driver"
-driver-version = "2.19"
+driver-long = "MongoDB .NET/C# driver"
+driver-short = ".NET/C# driver"
```
Comment (ccho-mongodb) [inline snooty.toml:22]: Question:
These seem like good changes to apply to all versions of the docs. Are these changes going to be backported? If so, it could be easier to create a separate PR for changes to backport so that the changes can be applied to each branch with minimal merge conflicts.

File: source/whats-new.txt [hunk 2]
```diff
@@ -12,10 +12,24 @@ What's New
...
+What's New in 1.3
+-----------------
+
+New features of the 1.3 {+product+} release include:
+
+- POCO to JSON analysis rules that show the serialization result of a POCO at compile
+  time.
+- POCO, LINQ, and Builders analysis support BSON serialization attributes.
+- Support for LINQ expressions that use enums in tuples. 
+- Improved support for nullable types.
```
Comment (ccho-mongodb) [inline source/whats-new.txt:31]: Suggestion:

I think all the list items are these are fragments rather than full sentences (perhaps the last one could be read either way), so they should not have periods at the end.

From the style guide on [list items](https://www.mongodb.com/docs/meta/style-guide/style/lists/#list-items):
>    -  In a list of only fragments, use no punctuation at the end of each item.

**Review by ccho-mongodb — APPROVED** [review]
LGTM! Left one question and suggestion.

File: snooty.toml (hunk 1 above)
Comment (mongoKart) [reply snooty.toml:22]: good call. i'll make those changes after i do this ticket.

File: source/whats-new.txt (hunk 2 above)
Comment (mongoKart) [reply source/whats-new.txt:31]: The second item is a full sentence.

Comment (ccho-mongodb) [reply source/whats-new.txt:31]: In that case, I would recommend changing all of them to be sentences to match this guidance from the same spot in the style guide:

> Make all list items parallel. For example, all items start with fragments, or all items use sentences. A list can have a mix of fragments and sentences as long as all of the items start with a fragment.
